Resources/Locale/en-US/tiles/tiles.ftl

```diff
@@ -87,3 +87,8 @@ tiles-flesh-floor = flesh floor
 tiles-techmaint3-floor = grated maintenance floor
 tiles-techmaint2-floor = steel maintenance floor
 tiles-wood2 = wood pattern floor
+tiles-desert-floor = desert floor
+tiles-low-desert-floor = low desert floor
+tiles-grass-planet-floor = grass planet floor
+tiles-basalt-floor = basalt floor
+tiles-snow-floor = snow floor
```

Comment:
Why are you adding new translation strings instead of replacing the existing translation strings?
I'm surprised this didn't error due to duplicates.

Reply:
I'm confused. I checked the file for those specific tile names and they didn't exist. So I assumed I had to add them, especially after @metalgearsloth said he forgot to add them to this file. 

Did I somehow get this wrong again ?